assets/js/components/KeyMetrics/widget-tiles/AnalyticsTrafficSourceWidget.test.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
We need to rename this file to match the parent file `AnalyticsTopTrafficSourceWidget.test.js` instead of `AnalyticsTrafficSourceWidget.test.js`

Reply:
Thanks, @eugene-manuilov. I have updated them.